oidc/provider.go

```diff
@@ -306,6 +307,18 @@ func (p *Provider) Exchange(ctx context.Context, oidcRequest Request, authorizat
 	if oidcRequest.PKCEVerifier() != nil {
 		authCodeOpts = append(authCodeOpts, oauth2.SetAuthURLParam("code_verifier", oidcRequest.PKCEVerifier().Verifier()))
 	}
+	if oidcRequest.ClientAssertionJWT() != nil {
+		// by now, sufficient validation should have already occurred to prevent
+		// errors here, but err check again just in case.
+		token, err := oidcRequest.ClientAssertionJWT().Serialize()
+		if err != nil {
+			return nil, err
```

Comment:
Perhaps follow the existing pattern and format for returned errors (incl. the `op`)

Reply:
apologies, I just missed it! my eyes might have started to glaze over after doing a couple dozen of em :stuck_out_tongue_closed_eyes: